### DISCUSSION TITLE
Red color for commits meaning?

### DISCUSSION THREAD

**Discussion by xfzv** [jesseduffield/lazygit#2207] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78810647/194357467-bf890d9c-bc36-4cc3-87f1-a2e89a3b5b10.png)

![image](https://user-images.githubusercontent.com/78810647/194357450-e5d0f67d-1d9b-4620-8024-0d5328f82fb9.png)

I guess the last two commits are in red because I'm on the main branch and said commits were merged from a different branch? Context: I made two commits on the `local` branch, switched to the `main` branch and ran `git merge local`.

Can someone please confirm/infirm?  Asking because these are the only two color settings for commits I could find in the doc:

https://github.com/jesseduffield/lazygit/blob/056cc1482168f0b7163325c1ebe20b6bcc6061ef/docs/Config.md?plain=1#L44-L47

The only occurrence for "red":

https://github.com/jesseduffield/lazygit/blob/056cc1482168f0b7163325c1ebe20b6bcc6061ef/docs/Config.md?plain=1#L48-L49

Thanks!

**Comment by mark2185** [comment, marked as answer]
I think those are the commits that differ from upstream.

**Reply by xfzv** [reply]
Ah, makes sense. They're not pushed indeed. So it has nothing to do with branches.
